Ashaba/API-Monitor
# Ensure all fields are sent

## What does this PR do?
Ensures all fields from the requests form are sent to the backend

## Tasks to be completed
- Ensure fields from request form *template* aren't sent
- Ensure fields that can have a single or many value(s) are always sent as arrays
- Rename keys for array fields to be plural

## How can this be tested
- Add a print statement to print received payload in the dashboard POST endpoint in `views.py`. Run the server.
- Visit the dashboard endpoint (e.g http://127.0.0.1:5000/dashboard), create some checks and click the save button.
- Verify that the payload printed on the server console contains all the submitted fields

## What is the relevant pivotal tracker story?
Story #155514650
